modules/site/src/app/examples/pagination/pagination-split-usage.component.scss

```diff
@@ -0,0 +1,63 @@
+@import "../../../ng2-material/core/style/variables";
+@import "../../../ng2-material/core/style/default-theme";
+@import "../../../ng2-material/components/whiteframe/whiteframe";
+
+pagination-split-usage {
```

Comment:
I need to check this out and try it to see the reason for this, but it seems like a lot of styling for an example - I would hope I could take a MD component and drop it in my view without having to override many styles.